python_modules/dagster/dagster/_core/instance/scheduling/__init__.py

```diff
@@ -0,0 +1,6 @@
+from dagster._core.instance.scheduling.scheduling_domain import SchedulingDomain
+
+__all__ = ["SchedulingDomain"]
+
+# For backward compatibility, provide scheduling_implementation as SchedulingDomain
+scheduling_implementation = SchedulingDomain
```

Comment:
this shouldn't be necessary right?

Reply:
will fix this (and look for any other similar issues) and add at the top of stack